docs/BreakingChanges/061 WebUtility.HtmlDecode, WebUtility.UrlDecode, and J.md

```diff
@@ -0,0 +1,32 @@
+## 61: WebUtility.HtmlDecode and WebUtility.UrlDecode methods
```

Comment:
{
  "Id": "61",
  "Title": "WebUtility.HtmlDecode and WebUtility.UrlDecode methods",
  "Details": "By default, decoding methods no longer decode an invalid input sequence into an invalid UTF-16 string. Instead, they return the original input.",
  "Suggestion": "The change in decoder output should matter only if you store binary data instead of UTF-16 data in strings. To explicitly control this behavior, set the aspnet:AllowRelaxedUnicodeDecoding attribute of the <a href=\"https://msdn.microsoft.com/en-us/library/ms228154(v=vs.110).aspx\">&lt;appSettings&gt;</a> element to true to enable legacy behavior or to false to enable the current behavior.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Net.WebUtility.HtmlDecode(System.String)",
    "M:System.Net.WebUtility.HtmlDecode(System.String,System.IO.TextWriter)",
    "M:System.Net.WebUtility.UrlDecode(System.String)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "This only matters if you store binary data instead of UTF-16 data in string. An analyzer may not add much beyond ApiPort, but we could do basic parameter analysis and potentially implement a code fix\nSource analyzer status: Pri 3, Planned",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Net.WebUtility.HtmlDecode(System.String)
- [x] M:System.Net.WebUtility.HtmlDecode(System.String,System.IO.TextWriter)
- [x] M:System.Net.WebUtility.UrlDecode(System.String)